site/docs/extensions/filter-control.md

```diff
@@ -254,6 +266,10 @@ Dependence if you use the datepicker option: [bootstrap-datepicker](https://gith
 
 * Clear all the controls added by this plugin (similar to showSearchClearButton option).
 
+### toggleFilterControl
+
+* Toggle filter control.
```

Comment:
Some more info would be greate e.g.   
Toggles the filter controls (Toggles the visibility of the filter controls)